.travis.yml

```diff
@@ -1,13 +1,34 @@
-language: swift
-os: osx
 osx_image: xcode10
-
+language: swift
+cache:
+  directories:
+  - Carthage
+env:
+  global:
+  - FRAMEWORK_NAME=MnemonicKit
+before_install:
+- gem install slather
+- brew update
+- brew outdated carthage || brew upgrade carthage
+- carthage bootstrap --platform iOS --cache-builds
+- pod install || pod install --repo-update
```

Comment:
This command now fails on CI: https://travis-ci.org/keefertaylor/MnemonicKit/builds/474772677

(Obviously there is no longer a Podfile)